installation/cmd/run.sh

```diff
@@ -4,14 +4,18 @@ set -o errexit
 
 ROOT_PATH=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )/../..
 defaultRelease=$(<"${ROOT_PATH}"/installation/resources/KYMA_VERSION)
-KYMA_RELEASE=${1:-$defaultRelease}
-INSTALLER_CR_PATH="${ROOT_PATH}"/installation/resources/installer-cr-kyma-diet.yaml
+KYMA_TAG=${1:-$defaultRelease}
```

Comment:
This name should be kept as KYMA_RELEASE in my opinion.

Reply:
Maybe we should create some small readme in `installation/resources` that describes some files (KYMA_VERSION and it's supported values, some CRs?)

Reply:
Will create it in another PR since we want to merge this asap